### ISSUE TITLE
failure installation on Mac OS 11.0.1 (Big Sur)

### ISSUE THREAD

**Issue by hoangducthuong** [astropy/pyregion#141]
I have a mac OS 11.0.1 Big Sur, python 3.9.
I am trying to install aplpy package, firstly I need to install pyregion requirement package. But the error is typical

pyregion/_region_filter.c:12082:52: error: no member named 'tp_print' in 'struct _typeobject'
      __pyx_type_8pyregion_14_region_filter_AngleRange.tp_print = 0;
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^
    14 errors generated.
    error: command '/usr/bin/gcc' failed with exit code 1
    ----------------------------------------
ERROR: Command errored out with exit status 1

Is this because of new Mac OS version?
Thanks for your help.

**Comment by bastianbeischer** [comment]
This because of python 3.9. I also had this problem with the released version of pyregion from pypi (2.0), but I think the problem is already fixed in the master branch, so you might want to try to install from github instead.

**Comment by leejjoon** [comment]
I believe this should be fixed with the v2.1.1 release. Please reopen if you still see the problem.
